README.md

```diff
@@ -32,6 +32,16 @@ repository instead of cloning it. See the [Customization guide](./docs/customiza
 If you want to build your own Flex marketplace by customizing the template application, see the
 [Customization guide](docs/customization-guide.md) documentation.
 
+## Deploying to Heroku
+
+**Note:** Remember to fork the repository before deploying the application. Otherwise there might be
+problems when trying to deploy new versions later!
```

Comment:
**Note:** Remember to fork the repository before deploying the application. Connecting your own Github repository to Heroku will make manual deploys easier.